README.md

```diff
@@ -21,14 +24,16 @@ This extension contributes the following additional settings:
 | `launchdarkly.streamUri`          |                         The LaunchDarkly stream uri to be used. Optional.                          | `https://stream.launchdarkly.com` |
 | `launchdarkly.enableHover`        |                  Enables flag info to be displayed on hover of a valid flag key.                   |                            `true` |
 | `launchdarkly.enableAutocomplete` |                                  Enable flag key autocompletion.                                   |                            `true` |
+| `launchdarkly.enableFlagExplorer`	|           Show all of the feature flags for the configured environment within the project.         |                            `true` |
 | `launchdarkly.sdkKey`             |      Your LaunchDarkly SDK key. OBSOLETE: Run the 'LaunchDarkly: Configure' command instead.       |                       `undefined` |
 | `launchdarkly.accessToken`        | Your LaunchDarkly API access token. DEPRECATED: Run the 'LaunchDarkly: Configure' command instead. |                       `undefined` |
-| `launchdarkly.enableFlagTreeview`	| 			Show all of the feature flags for the configured environment within the project.		 |							  `true` |
 
-**Note:** If you use quick suggestions to autocomplete words, LaunchDarkly autocomplete functionality requires the `editor.quickSuggestions.strings` setting to be enabled. Otherwise, you'll need to press `Ctrl+Space` (default binding) to see your flag key suggestions.
+
 
 Here's an example configuration with quick suggestions enabled:
 
+**Note:** If you use quick suggestions to autocomplete words, LaunchDarkly autocomplete functionality requires the `editor.quickSuggestions.strings` setting to be enabled. Otherwise, you'll need to press `Ctrl+Space` (default binding) to see your flag key suggestions.
```

Comment:
Oops, looks like this note got reordered - can we move it back up above to the previous line of text?